### DISCUSSION TITLE
Improved vector editor ( blurred background, shadow )

### DISCUSSION THREAD

**Discussion by ApplikationFree** [ScratchAddons/ScratchAddons#3321] [Ideas]
The Scratch vector editor is already very complete while being simple but changes could be made to the editor.
**The shadows**
The shadows allow to give the impression that an object is in hover, it is supported by the SVG format but not by the vector editor of Scratch,
![image](https://user-images.githubusercontent.com/85065664/128622777-a7ca9846-c03c-4864-a157-af5ec820272a.png)

![image](https://user-images.githubusercontent.com/85065664/128622773-d8165dd5-e854-400e-83da-4b760c15355e.png)

**Comment by lisa-wolfgang** [comment]
I don't believe that the Scratch renderer (not just the vector editor) supports blur. Not sure about shadows...

**Reply by williamhorning** [reply]
> I don't believe that the Scratch renderer (not just the vector editor) supports blur. Not sure about shadows...

Any CSS can be applied to SVGs, this includes the CSS blur effect.

**Reply by lisa-wolfgang** [reply]
It's possible that I wasn't doing it correctly, but when I applied blur to an SVG and imported it, the blur effect was not present.

**Reply by infinotiver** [reply]
I think opacity is a very good Idea , please make it atleast because I am making few projects in which I want some part of costume to be transparent but as there is no option , I have to use cloning

**Reply by Samq64** [reply]
> I think opacity is a very good Idea , please make it atleast because I am making few projects in which I want some part of costume to be transparent but as there is no option , I have to use cloning

Make your costume in TurboWarp and then import it into Scratch.

**Reply by nexensys** [reply]
> Not sure about shadows... Shadows appear to be supported, but I would ask @TheColaber.

**Reply by TheColaber** [reply]
It is indeed not supported but the canvas does render all svgs normally
![image](https://user-images.githubusercontent.com/72760579/135182670-d15097ba-874a-4687-9f2e-94315ffc65ff.png)
